user/enterprise/platform-tips.md

```diff
@@ -254,3 +254,15 @@ After that, scroll down and click "Save". After your changes have been saved, yo
 ```
 $ replicatedctl app start
 ```
+
+Let's Encrypt certificates are short-lived, this means they expire after 90 days. This means that you'll have to renew them on a regular basis. Thankfully this can be done with `certbot` as well. Run the following commands in order to renew your certificate.
+
+**Note: This will also cause downtime**
+
+```
+$ replicatedctl app stop
+$ sudo certbot renew
+$ replicatedctl app start
+```
+
+In general: These certificate renewals should be automated with a cron job.
```

Comment:
You ended up not going with the pre and post hooks?

Reply:
No, I decided to do it like this for now which makes it a bit more obvious what's going on. I didn't feel a 100% comfortable about the pre and post hooks yet.

Reply:
👍 That makes sense.